docs/react-v9/contributing/perf-testing.md

```diff
@@ -0,0 +1,215 @@
+Performance testing with flamegraphs is a feature that runs on all PRs opened against Fluent UI React and was introduced with [PR #9550](https://github.com/microsoft/fluentui/pull/9550). This page provides an overview of this feature. This process is highly recommended, but no longer a hard requirement to check work in.
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31567&commitId=2f15ad3bd3b522fe921745d967b9628846540c58&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|1| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|7| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>